src/main/ruby/truffleruby/core/process.rb

```diff
@@ -400,10 +400,18 @@ def self.gid=(gid)
   end
 
   def self.euid=(uid)
+    require 'etc'
```

Comment:
This can be done inside the `if uid.kind_of?(String)` branch, so it's not required if passed an integer.